src/ua.skrypnyk.hw10/Circle.java

```diff
@@ -0,0 +1,48 @@
+package src.ua.skrypnyk.hw10;
+
+import java.util.Scanner;
+
+public class Circle extends Shape {
+    private double perimeter;
+    private double area;
+    private int radius;
+
+    public Circle(double perimeter, double area) {
```

Comment:
Are and perimeter have to be calculated values
You do not need to pass them to constructor 

public Circle(double radius) {}
You should initialize radius via constructor